src/styles/components/_override.buttons.scss

```diff
@@ -81,44 +111,51 @@
   &:focus,
   &.ds-c-button--focus {
     border-color: $focus-choice-color;
+    color: $color-white;
   }
 
-  // Teal (primary) transparent button
-  &.ds-c-button--transparent {
-    // TODO: Fix transparent button styles after core CMSDS improves their buttons styles
-    color: $color-primary;
-    // without !important, links that extend transparent and primary get the
-    // normal primary button dark teal background on hover
-    background-color: transparent !important;
-    border-color: transparent;
+  // Primary white solid inverse button
+  &.ds-c-button--inverse,
+  &.ds-c-button--inverse:visited {
+    background-color: $color-white;
+    border-color: $color-gray-dark;
+    color: $color-gray-dark;
 
     &:hover,
     &.ds-c-button--hover {
-      color: $color-primary-dark;
+      border-color: rgba($color-gray-dark, 0.8);
+      color: rgba($color-gray-dark, 0.8);
     }
-  }
-}
 
-// Teal (primary) outline button
-.ds-c-button--primary-outline,
-.ds-c-button--primary-outline:visited {
-  border-color: $color-primary;
-  color: $color-primary;
+    &:focus,
+    &.ds-c-button--focus {
+      border-color: rgba($color-gray-dark, 0.8);
```

Comment:
Should the inverse primary focus state have any additional styles? In the normal focus state the color and border color stay the same

Reply:
That is a good question. I modeled it after the main DS where the inverse primary focus state's color and border match the hover state's color and border but the non-inverse one matches the normal state, so maybe we want to keep with that. @babsmiles what are your thoughts?